Sources/Player/OfflineEngine/Internal/Storage/GRDBStorage/GRDBOfflineStorage.swift

```diff
@@ -122,7 +117,22 @@ private extension GRDBOfflineStorage {
 	static func databaseURL() throws -> URL {
 		let appSupportURL = PlayerWorld.fileManagerClient.applicationSupportDirectory()
 		let directoryURL = appSupportURL.appendingPathComponent("PlayerOfflineDatabase", isDirectory: true)
-		try PlayerWorld.fileManagerClient.createDirectory(at: directoryURL, withIntermediateDirectories: true)
+		if PlayerWorld.fileManagerClient.fileExists(atPath: directoryURL.path, isDirectory: nil) {
+			var attributes = try? PlayerWorld.fileManagerClient.attributesOfItem(directoryURL.path)
+			if attributes != nil,
+			   let protectionKey = attributes?[FileAttributeKey.protectionKey] as? FileProtectionType,
+			   protectionKey != FileProtectionType.none
+			{
+				attributes?[FileAttributeKey.protectionKey] = FileProtectionType.none
+				try? PlayerWorld.fileManagerClient.setAttributes(attributes!, directoryURL.path)
```

Comment:
maybe log this as well if throws?

Reply:
yeah. I might combine both in a single log. No need to differentiate really (especially if I attach the error description).